src/lib/nifindexes.nim

```diff
@@ -155,17 +158,39 @@ proc createIndex*(infile: string; buildChecksum: bool) =
   content.add toString(public)
   content.add "\n"
   content.add toString(private)
+  content.add "\n"
+
+  for (key, values) in hookIndexMap.pairs:
+    var hookSectionBuf = default(TokenBuf)
+    let tag = registerTag(key)
+    hookSectionBuf.addParLe tag
+
+    for value in values:
+      let (obj, sym) = value
+      let diff = symToOffsetMap[sym] - symToOffsetMap[obj]
```

Comment:
That's not how these diffs work. But you can just store this offset as it is without any diffing this time.